### ISSUE TITLE
Methods of retraining and testing using models

### ISSUE THREAD

**Issue by yanpinlu** [jiangdada1221/TEINet#1]
Thank you very much for designing such a clever negative sample selection strategy, I have the following doubts when using it:
1. Can it be understood that when using the model training, only the positive data set needs to be input and the test set organized according to different strategies? And use the following code for training and prediction? (python train.py --train_file data/train_pos.csv --test_file data/test.csv --epochs 30 --model_path results/model.pth
)
2. Do I need to re-run TCRpeg to generate embedded TCR and epitope information when using the new data set for model training and evaluation? @jiangdada1221

**Comment by jiangdada1221** [comment]
Dear yanpinlu,

For the first question, only the positive data of the **training set** needs to be input. The test set contains both the positive and sampled negative data since we need both of them to compute the AUC value. You can download the data and see the training and validation dataset in the 5fold crossvalidation.

For the second one, you don't need to re-run TCRpeg. TCRpeg was pretrained on a large TCR pool. So you can just use the pretrained TCRpeg deposited in this repository.
@yanpinlu
